src/amo/css/App.scss

```diff
@@ -27,7 +27,24 @@ body {
 }
 
 .App-content {
-  padding: 0 0 10px;
+  background: $accent-color;
+
+  > div {
+    padding: 10px;
+  }
+
+  > .full-width {
```

Comment:
It's weird because for most pages it's nice not having to add the padding to all the content, but then add-ons details wants to be full-width.

Fair enough though, I'll take these out.